foundryup/foundryup

```diff
@@ -240,7 +251,9 @@ USAGE:
 
 OPTIONS:
     -h, --help      Print help information
-    -v, --version   Install a specific version from built binaries
+    -i, --install   Install a specific version from built binaries
```

Comment:
nits:

not a regression but running `--install` without comments silently fails (this is expected but ideally should raise an error):

```
foundryup --install
```

ideally `--use` without an argument should display `foundryup: version not installed`, right now it panics with `cp: cannot stat '/home/zerosnacks/.foundry/versions//forge': No such file or directory`

./foundryup --use